code/api/queries.md

```diff
@@ -0,0 +1,51 @@
+Get all user objects:
+
```

Comment:
This file could be added to git ignore before deployment, thoughts?

Reply:
Yeah, agreed. It's really there as a resource for us in developing/testing in GraphiQL. Doesn't actually add any value to code base.